cms/static/js/i18n/ar/djangojs.js

```diff
@@ -543,7 +543,7 @@
     "Create an Account": "\u0625\u0646\u0634\u0627\u0621 \u062d\u0633\u0627\u0628", 
     "Create an Account.": "\u0625\u0646\u0634\u0627\u0621 \u062d\u0633\u0627\u0628.", 
     "Create an account using": "\u0623\u0646\u0634\u0626 \u062d\u0633\u0627\u0628\u0627\u064b \u0628\u0627\u0633\u062a\u062e\u062f\u0627\u0645", 
-    "Create an {platform} account": "\u0625\u0646\u0634\u0627\u0621 \u062d\u0633\u0627\u0628 {platform}", 
```

Comment:
I think this was correct. We should never translate placeholders like `{platform}`.

Reply:
Translation reverted.